app/util.py

```diff
@@ -76,26 +76,22 @@ def is_logged_in():
     return True
 
 
-def error_message(array):
-    if type(array) is dict:
-        if 'error' in array:
-            flash("Error: " + array["error"] + " - " + array["text"])
-            # There was an error
-            return True
-        elif 'pymesync error' in array:
-            flash("Error: " + array["pymesync error"] + " - " + array["text"])
-            # There was an error
-            return True
-    elif type(array) is list:
-        if 'error' in array[0]:
-            flash("Error: " + array[0]["error"] + " - " + array[0]["text"])
-            # There was an error
-            return True
-        elif 'pymesync error' in array:
-            flash("Error: " + array[0]["pymesync error"] + " - " +
-                  array[0]["text"])
-            # There was an error
-            return True
+def error_message(obj):
+    # obj is empty, there was an error
+    if not obj:
+        return True
```

Comment:
Empty values aren't necessarily error values. This should return False instead of True